### DISCUSSION TITLE
Flows - Operation Read Data - Directus collection

### DISCUSSION THREAD

**Discussion by gkielwasser** [directus/directus#14362] [Archive]
Hi,

I'm not sure it an issue or it has been done on purpose.

Why the read data operation cannot deals with Directus collections ? it seems to be filtered/excluded.

One simple use case would be to read additionnal data about an user to retrieve its email. We would need to access `directus_users` table.

Thanks

**Comment by rijkvanzanten** [comment]
I'm pretty sure there's another open discussion/issue about this, but I can't seem to find it 🤔 

Anyways, it's because the system collections should be accessed through dedicated Services, to make sure that all core logic is executed as well (users for example have password checks, fields alter the schema, etc). The operation is currently relying on the ItemsService exclusively, so it's only safe to use against regular collections 👍🏻

**Reply by gkielwasser** [reply]
Ok I 'm curently having a flow that is sending an email to the user who has placed an order. 
Once an order is created it trigger a transactionnal email to him.
For now I'm relying on the O2M relation there is between my orders collection and the directus_users collection in order to retrieve the email of my user.
So I retrieve again the order and I read deeply the user property to query the email.
I guess that's the way to go :)

**Reply by rijkvanzanten** [reply]
That should do the trick for now!

**Comment by izoukhai** [comment]
You still can access system collections, you have to write the collection name directly using raw value
<img width="846" alt="Capture d’écran 2022-07-11 à 15 14 59" src="https://user-images.githubusercontent.com/44984965/178272597-8afc9dc2-0f96-432a-9446-c06dc85f536d.png">
<img width="846" alt="Capture d’écran 2022-07-11 à 15 15 13" src="https://user-images.githubusercontent.com/44984965/178272644-cf5db4ee-fbf7-430e-bfe0-acae364b9e6f.png">

**Reply by gkielwasser** [reply]
> You still can access system collections, you have to write the collection name directly using raw value <img alt="Capture d’écran 2022-07-11 à 15 14 59" width="846" src="https://user-images.githubusercontent.com/44984965/178272597-8afc9dc2-0f96-432a-9446-c06dc85f536d.png"> <img alt="Capture d’écran 2022-07-11 à 15 15 13" width="846" src="https://user-images.githubusercontent.com/44984965/178272644-cf5db4ee-fbf7-430e-bfe0-acae364b9e6f.png">

@izoukhai Thanks a lot ! 😃

**Reply by Mehdi-YC** [reply]
Hi thank you, this is helpful, can this be done on a M2M field ? for example field.price, to get the listofprices in many to many field ?
